dataset/gedi_rasterization.py

```diff
@@ -0,0 +1,103 @@
+
+import ee
+import json
+import os
+import datetime
+import time
+from ee.batch import Export
+import ast
+
+def make_timestampFromYMD(year, month, day):
+    tdat = datetime.datetime(year, month, day)
+    return time.mktime(tdat.timetuple()) * 1000
+
+def rasterize_GEDI_by_utm_zone(orbits, grid_id, month):
+    props = ["orbit", "track", "beam", "channel", "degrade", "dtime", "quality",
```

Comment:
could you briefly comment why these properties were selected?

Reply:
"orbit", "track", "beam", "channel" are needed to keep the data identity. 

"degrade", "dtime", "quality" are quality flags that were identify as quality flag for L4A and L4B biomass modeling.